Maquetado/index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="application.css">
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <img src="assets/wolox_logo.svg" alt="wolox" />
+      <h1 class="main-title">
+        Developer Tools
+      </h1>
+    </nav>
+
+    <div class="main-content">
+      <div class="container-techno">
+        <h1 class="type-tool">
+          Front-End Tools
+        </h1>
+        <div class="tools-container">
+          <div class="techno">
+            <img src="assets/react_icon.svg" alt= "React"  class="logo-techn" >
```

Comment:
Remember to not leave a space between the attribute and it's value, check the whole file :smile: